content/guides/continuous-integration/github-actions.md

```diff
@@ -454,6 +454,73 @@ file.
 
 <DocsImage src="/img/guides/github-actions/rwa-run-matrix.png" alt="Cypress Real World App GitHub Actions Matrix"></DocsImage>
 
+## Common Problems and Solutions
+
+### Re-run jobs passing with empty tests
+
+We recommend passing the `GITHUB_TOKEN` secret (created by the GH Action
+automatically) as an environment variable. This will allow the accurate
+identification of each build to avoid confusion when re-running a build.
+
+```
+name: Cypress tests
+on: [push]
+jobs:
+  cypress-run:
+    name: Cypress run
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          record: true
+        env:
+          # pass GitHub token to detect new build vs re-run build
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+```
+
+### Pull requests commit message is `merge SHA into SHA`
+
+You can overwrite the commit message sent to the Dashboard by setting an
+environment variable. See
+[Issue #124](https://github.com/cypress-io/github-action/issues/124) for more
+details.
+
+```
+name: Cypress tests
+on: [push]
+jobs:
+  cypress-run:
+    name: Cypress run
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          record: true
+        env:
+          # overwrite commit message sent to Dashboard
+          COMMIT_INFO_MESSAGE: ${{github.event.pull_request.title}}
+          # re-enable PR comment bot
+          COMMIT_INFO_SHA: ${{github.event.pull_request.head.sha}}
+```
+
+<Alert type="warning">
+
+We also recommend adding `COMMIT_INFO_SHA` to re-enable
+[Cypress bot PR comments](https://on.cypress.io/github-integration#Pull-request-comments).
+See
+[this comment](https://github.com/cypress-io/github-action/issues/124#issuecomment-716584972)
```

Comment:
Is this the correct comment URL? Seem like it's pointing to someone asking a question about Gleb's solution breaking other stuff. I wasn't sure if that's the intent here. 

Also, and this is subtle, but the alerts have different classes that indicate some level of context. The yellow is a warning. Like in https://deploy-preview-4488--cypress-docs.netlify.app/guides/continuous-integration/github-actions#Setting-up-Parallelization where give explicit steps to avoid issues. This may be more of 'info' or 'success' messaging depending on the intent/link of the above.

Reply:
Adding the `COMMIT_INFO_MESSAGE` env breaks the PR bot. Removing the env var no longer fixes the issue because GitHub never receives the final SHA event.